pkg/config/config.go

```diff
@@ -28,6 +30,8 @@ func Get() Config {
 			AllowOrigins:     getEnvSlice("ACCESS_CONTROL_ALLOW_ORIGIN", []string{"*"}, ","),
 			AllowCredentials: getEnvBool("ACCESS_CONTROL_ALLOW_CREDENTIALS", false),
 		},
+		StaticFilesDir:  getEnv("STATIC_FILES_DIR", ""),
+		ApiRoutePrefix: "/api", //should start with a "/",
```

Comment:
I'm wondering should this perhaps be `api/v1` to version the api?

Reply:
Yeah it definitely should. It could even be separated into a separate config variable for maximum flexibility.

```go
...
ApiRoutePrefix: "/api", //should start with a "/",
ApiVersionPrefix: "/v1"
...
```

Reply:
Agree - done